conf/modules.config

```diff
@@ -700,6 +700,70 @@ process {
         ext.args = '--mitochondria-mode TRUE'
         }
 }
+//
+// ANALYSE_MT:ALIGN_AND_CALL_MT_SHIFT
+//
+process {
+    withName: '.*ANALYSE_MT:PREPARE_GENOME_MT:PREPARE_GENOME:BWAMEM2_INDEX' {
+        ext.when = {!params.bwamem2_index && params.aligner == "bwamem2"}
+        publishDir = [
+            path: { "${params.outdir}/references" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+            enabled: params.save_reference,
+            pattern: "*{0123,amb,ann,bwt.2bit.64,bwt.8bit.32,pac}"
+        ]
+    }
+
+
```

Comment:
I think the general standard is to leave just one line between two modules here to help with readability. So my suggestion would be to delete the extra empty lines here.